aptos-move/framework/src/natives/code.rs

```diff
@@ -192,11 +192,35 @@ const EALREADY_REQUESTED: u64 = 0x03_0000;
 const ARBITRARY_POLICY: u8 = 0;
 
 /// The native code context.
-#[derive(Tid, Default)]
+#[derive(Tid)]
 pub struct NativeCodeContext {
+    /// If false, publish requests are ignored and any attempts to publish code result in runtime
+    /// errors.
+    enabled: bool,
     /// Remembers whether the publishing of a module bundle was requested during transaction
```

Comment:
This comment seems rather confusing, first if it *remembers* anything, it's not whether, it's the actual request.
If we want to, there are more useful things a comment could address: how is it used? can a request come when another request is already stored? etc

Reply:
Updated.